turbopack/crates/turbo-tasks-memory/src/task.rs

```diff
@@ -1621,17 +1621,15 @@ impl Task {
 
     pub(crate) fn get_or_wait_output<T, F: FnOnce(&mut Output) -> Result<T>>(
         &self,
-        strongly_consistent: bool,
+        consistency: ReadConsistency,
         func: F,
         note: impl Fn() -> String + Sync + Send + 'static,
         backend: &MemoryBackend,
         turbo_tasks: &dyn TurboTasksBackendApi<MemoryBackend>,
     ) -> Result<Result<T, EventListener>> {
         let mut aggregation_context = TaskAggregationContext::new(turbo_tasks, backend);
-        if strongly_consistent {
+        let mut state = if consistency == ReadConsistency::Strong {
             prepare_aggregation_data(&aggregation_context, &self.id);
```

Comment:
Actually this line can could be removed. `aggregation_data` and `prepare_aggregation_data` does the same